process_collections.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
Usually when I write ruby script that ends up shelling out to start another ruby script, I take that to mean it should have been a shell script all along. That's the feeling I get here.

Reply:
I am hopeless at doing anything other than copy-pasting shell. So yeah. Anything I can put in a Ruby string and invoke with backticks, I'm there.